### ISSUE TITLE
Fix the Ores（Thorium and Ore dictionaries）

### ISSUE THREAD

**Issue by tong-ge** [ZelGimi/industrialupgrade#30]
In real life,thorium is a metal which emits primarily alpha rays,which can't even go through a paper.But in this mod,thorium ores drop thorium crystals which does a significance harm to the player.Thorium can't be used directly in a reactor,If you want to use thorium in a reactor,you must start it with uranium or plutonium.And all the three ores in this mod have no oredicts,so they have poor compatibilities.Platinum is even useless!

**Comment by aking4125** [comment]
In the config. The ores cannot be disabled or adjusted. Platinum is completely useless in this mod. There should be an oreDict with Platinum and Iridium ores. Thorium should also be an ingot and also oreDict with many other mods ingots. 

The Particle Converter should also accept any Iridium ore as an input. It's not reading the Iridium oreDict. For example, Thermal Expansions Iridium ore should be accepted in the Particle converter as the ingots from Thermal already are. 

In short, the ores and their products should be oreDict. The config should give an option to enable or disable the ore generation. 

Platinum is completely useless.

**Comment by ZelGimi** [comment]
In the new version 1.12.2 2.0, all ingots and ores will be needed and will have OreDictionary

**Comment by tong-ge** [comment]
But till now,2.x has only 1.7.10------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年9月18日(星期六) 凌晨4:08
***@***.***&gt;;
抄送:&nbsp;&quot;Bruce ***@***.******@***.***&gt;;
主题:&nbsp;Re: [ZelGimi/industrialupgrade] Fix the Ores（Thorium and Ore dictionaries） (#30)
